src/Proto3/Suite/DotProto/Generate.hs

```diff
@@ -546,6 +555,67 @@ messageInstD ctxt parentIdent msgIdent messageParts =
                       , decodeMessageDecl
                       , dotProtoDecl ]))
 
+toJSONPBMessageInstD :: TypeContext -> DotProtoIdentifier -> DotProtoIdentifier
+                     -> [DotProtoMessagePart] -> CompileResult HsDecl
+toJSONPBMessageInstD _ctxt parentIdent msgIdent messageParts =
```

Comment:
I'm not sure how you feel about this so it's up to your judgement, but I only _know_ what the generated code mostly looks like from reading the PR description. Which leads me to think a comment providing a very limited example of an out-of-order field number-tagged to instance will be helpful for posterity and maintenance.

(same comment for `fromJSONPBMessageInstD`.